e2e/libs/volume/base.py

```diff
@@ -1,8 +1,12 @@
 from abc import ABC, abstractmethod
+from utility.utility import set_annotation
+from utility.utility import get_annotation_value
 
 
 class Base(ABC):
 
+    ANNOT_DATA_CHECKSUM = f'test.longhorn.io/data-checksum-'
```

Comment:
Why not reuse [ANNOT_CHECKSUM](https://github.com/longhorn/longhorn-tests/blob/297533fc8894a203f1777b304c25c2f2119771c6/e2e/libs/utility/constant.py#L7)?

Reply:
`ANNOT_CHECKSUM` is the last recorded, but now every checksum of every write has to be recorded.

Reply:
I see, could you move this to a relevent constant file?

Reply:
Refined. Thank you!